src/pages/editProfiles/utils/inputUtils.ts

```diff
@@ -0,0 +1,10 @@
+export const allowOnlyNumberKey = (e: React.KeyboardEvent<HTMLInputElement>) => {
+  const isNumber =
+    (e.key >= '0' && e.key <= '9') || ['Backspace', 'Delete', 'ArrowLeft', 'ArrowRight', 'Tab'].includes(e.key);
+  if (!isNumber) e.preventDefault();
+};
+
+export const allowOnlyNumberPaste = (e: React.ClipboardEvent<HTMLInputElement>) => {
+  const paste = e.clipboardData.getData('text');
+  if (!/^\d+$/.test(paste)) e.preventDefault();
+};
```

Comment:
```suggestion
import type { KeyboardEvent, ClipboardEvent } from 'react';

export const allowOnlyNumberKey = (e: KeyboardEvent<HTMLInputElement>) => {
  const isNumber =
    (e.key >= '0' && e.key <= '9') || ['Backspace', 'Delete', 'ArrowLeft', 'ArrowRight', 'Tab'].includes(e.key);
  if (!isNumber) e.preventDefault();
};

export const allowOnlyNumberPaste = (e: ClipboardEvent<HTMLInputElement>) => {
  const paste = e.clipboardData.getData('text');
  if (!/^\d+$/.test(paste)) e.preventDefault();
};
```

다음과 같이 타입을 명시적으로 임포트해서 사용하는 것이 어떨까요?? 해당 부분을 컨벤션을 안맞추어서 이야기 해봐야될 것 같긴한데.
`import type`은 번들링 시 제거되므로 최종 번들 크기를 줄일 수 있어 최적화 측면에서도 유리하고, TypeScript 공식문서에서도 추천하고 있는 방식입니다. 관련 링크 남길게요!

https://www.typescriptlang.org/ko/docs/handbook/release-notes/typescript-3-8.html

Reply:
오 !! 많이 배웁니다 !